### ISSUE TITLE
UP Commons journal access question - can we help them facilitate this access? 

### ISSUE THREAD

**Issue by bjr70** [mlaa/commons#863]
**tl;dr**

Brenna McLaughlin from AUPresses (Research and Communications Director) is asking if we can facilitate this helping the University of Toronto give every member of AUPresses access to one of their journals. The journal is not open access, so remains behind a paywall. UofT has suggested two methods I recognize as commonly used to facilitate institutional access to paywalled journals, but this is a much different situation.

**Message from Brenna**

Essentially, the University of Toronto Press wants to give our membership free access to one of their journals. The journal is not otherwise generally OA, so the access needs to be managed through our password-protection system through which we provide access to all other member-only content. BUT, of course, this is all Commons-based now, not our own website, so I am not convinced we can set it up the way they envision. They've only talked about this with not-Kate-or-me staff members, so it's been a little bit a game of telephone to figure out what is wanted but here's what they are outlining (*my additional notes bracketed with asterisks*):
1.	Website IP
•	Find the resources.up.hcommons.org site Wordpress IP: Log in to your WordPress hosting account and look for the server information section. (*AUPresses does not have access to the server/IP information as far as I can tell.*)
•	URL for error page (if a user fails to authenticate/gain access) (*Assuming this is asking about  when a person doesn't have or isn't logged into an active UPC account. But I think this is a general HC registry page?*)
Then they propose that AUPresses will:
1.	Add PHP to the AUPresses resources site.
•	(*pretty sure we don't have this kind of access?*) To add a PHP snippet to Wordpress: Use a "code snippet" plugin or go to Appearance -> Theme File Editor -> *custom* (NOT other areas, as theme updates will override) -> custom_functions.php
•	In the user portal (*I am not sure what they mean by user portal! Perhaps landing page for the journal access benefit?*), add the text provided by Antonia with a link users can click to the journal: https://www.utpjournals.press/loi/jsp

**Comment by bjr70** [comment]
Initial discussion with Mike indicates that the two methods suggested are not workable. There are two possible solutions that both have drawbacks and would take time. Looping Zoe in for more discussion.

**Comment by bjr70** [comment]
They've decided to go another route.
